tests/specs/guardian-redirects.spec.js

```diff
@@ -198,7 +198,7 @@ testScenarios.forEach((scenario) => {
       test(`Verify redirect for ${baseUrl}/r/vpn/update/ios, C1539669`, async () => {
         await verifyRedirectUrl(
           `${baseUrl}/r/vpn/update/ios`,
-          'https://apps.apple.com/us/app/firefox-private-network-vpn/id1489407738'
+          'https://apps.apple.com/us/app/mozilla-vpn-secure-private/id1489407738'
```

Comment:
I'm not sure when this changed, but it was not a Guardian change -- the former is now a redirect to the latter (and redirect tests here assert against the last redirect URL in the chain). I would presume this changed as part of the FPN removal.